### ISSUE TITLE
Feature request: Example lists

### ISSUE THREAD

**Issue by kernie** [quarto-dev/quarto#91]
Hi,

I really like the visual mode in vscode!

However, in one document I get this message
<img width="307" alt="Bildschirm­foto 2023-02-17 um 16 09 04" src="https://user-images.githubusercontent.com/23321751/219692266-5ee3f109-7436-4cab-9659-272c969022c3.png">

Are there plans to extend the functionality?

**Comment by mcanouil** [comment]
The message states "not currently supported" and not "will never be supported".
So in due time, it should be supported, just be patient. 😉

**Comment by jjallaire** [comment]
Yes, for this to be supported we need some changes in the Pandoc AST.
